### DISCUSSION TITLE
Difference in Performance score of Pagespeed Insights vs. Lighthouse in Google Chrome Dev Tools

### DISCUSSION THREAD

**Discussion by dontwakeme** [GoogleChrome/lighthouse#15663] [Q&A]
Today me and a collague did a lot of pagespeed tests on various websites both on Pagespeed Insights ( https://pagespeed.web.dev/ ) and also in the Chrome Dev Tools using incognito mode.
In nearly all tests the performance score of lab data was way higher (20 to 25 points) when using chrome dev tools. 
I included some screenshots to give some examples (some of them in german but I guess one should be able to still see the difference) Me and a colleague (both on different computers and different networks) could replicate that is on a lof of different websites as well.

From what i understand PSI labdata is also using lighthouse, so how come there is such a big difference in the performance score?
Is this just the different version? PSI states it uses 10.0.0 and Chrome Dev Tools 10.1.0
Or did I get something totally wrong?

Thx and looking forward to your answers!
google.com lighthouse 

![google-lighthouse](https://github.com/GoogleChrome/lighthouse/assets/26796678/44547a0c-383c-4763-9ad1-54b94fb27480)

google.com PSI
![google-psi](https://github.com/GoogleChrome/lighthouse/assets/26796678/a28d15b9-e602-4819-9605-a6e9fef50ad4)

krone.at lighthouse
![krone-lighthouse](https://github.com/GoogleChrome/lighthouse/assets/26796678/8d0a5d3e-c1ed-45d3-8862-5a79ee07333b)

krone.at PSI
![krone-PSI](https://github.com/GoogleChrome/lighthouse/assets/26796678/df08c6b3-d2d8-45b8-ba0e-a2f00ed432ea)

**Comment by adamraine** [comment]
https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md
https://www.debugbear.com/blog/why-is-my-lighthouse-score-different-from-pagespeed-insights

**Comment by JuanRomeroDominguez** [comment]
Everything affects. You must take into account the speed of your Internet, CPU, whether or not you have ads, etc... When I am optimizing the [calendar](https://calendario.gratis/en-at/gregorian/2024/) website that I have, I choose the same characteristics that appear in pagespeed in the chrome browser and that is where I am testing .

Greetings.
